checks.d/sqlserver.py

```diff
@@ -185,19 +186,21 @@ def get_cursor(self, instance):
             try:
                 conn_str = self._conn_string(instance)
                 conn = adodbapi.connect(conn_str)
+                conn.CommandTimeout = int(instance.get('command_timeout',
+                                                        self.DEFAULT_COMMAND_TIMEOUT))
```

Comment:
`conn.CommandTimeout` was set to 30seconds by default. As you are changing it to 5 seconds now, it may break some users configuration ? What do you think ?